ldk/javascript/src/types/oliveHelps/whisper.d.ts

```diff
@@ -113,7 +113,17 @@ declare namespace WhisperService {
     value: string;
   };
 
+  interface AutocompleteFilterOptions {
+    ignoreAccents?: boolean;
+    ignoreCase?: boolean;
+    limit?: number;
+    matchFrom?: 'any' | 'start';
+    stringify?: (option: AutocompleteOption) => void;
```

Comment:
Based on the component's code, I think stringify should be required here, or made optional in the UI

```js
  const parsedFilterOptions = filterOptions ? createFilterOptions({
    ...filterOptions,
    stringify: option => filterOptions.stringify(option),
  }) : undefined;
```